app/src/main/java/com/merahputihperkasa/prodigi/utils/Utils.kt

```
@@ -22,14 +25,26 @@ fun isValidURi(uri: String): Boolean {
+        Context.CLIPBOARD_SERVICE
+    ) as ClipboardManager
     val clip = ClipData.newPlainText(text, text)
     clipboard.setPrimaryClip(clip)
 }
 
 fun openUrl(context: Context, url: String) {
     val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
-    context.startActivity(intent)
+    try {
+        context.startActivity(intent)
+    } catch (e: ActivityNotFoundException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)